src/Manifest/Manifest.router.ts

```diff
@@ -77,12 +87,34 @@ export class ManifestRouter extends Router {
 
   async getProjectManifest(req: AuthRequest) {
     const id = server.extractFromReq(req, 'id')
+
     const body = await new S3Project(id).readFileBody(MANIFEST_FILENAME)
     if (body) {
       return JSON.parse(body.toString())
     }
   }
 
+  async getManifests(req: AuthRequest) {
+    let eth_address = req.auth.ethAddress
+
+    const projectSearcher = new SearchableProject(req)
+    const projects = await projectSearcher.searchByEthAddress(eth_address)
+    let manifests: any[] = []
```

Comment:
I think juanca touched on this on his comment but I'd change this from any to the actual type, is it ManifestAttributes? (from Manifest.types.ts)